chat/public/js/src/chat.js

```diff
@@ -361,9 +361,8 @@ function ChatXBlock(runtime, element, init_data) {
      * pause: delays execution with a timeout
      */
     var pause = function(timeout) {
-        return new Promise(function(resolve, reject) {
-            setTimeout(resolve, timeout);
-        });
+        var promise = $.Deferred();
+        setTimeout(promise.resolve, timeout);
```

Comment:
Try cherry-picking this commit from my PR: https://github.com/open-craft/xblock-chat/pull/12/commits/c2a320c7c5f5c8fa03ab63b254835e7252b1a190

That could fix the failing test.